test/parallel/test-fs-write-stream-file-handle.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+const tmpdir = require('../common/tmpdir');
+const file = path.join(tmpdir.path, '/write_stream_filehandle_test.txt');
```

Comment:
```suggestion
const file = path.join(tmpdir.path, 'write_stream_filehandle_test.txt');
```

Reply:
The problem was that I was calling `tmp.refresh()` in both threads after the thread creation, but you are right